src/App.vue

```diff
@@ -228,13 +225,13 @@ export default {
 				return
 			}
 			this.loading.create = true
-			createNote(this.filter.category || '')
+			createNote(this.filter.category)
 				.then(note => {
-					this.routeToNote(note.id, { new: null })
+					this.routeToNote(note.id)
```

Comment:
Why do you remove the parameter `new`? It has the purpose, that the note's title is automatically generated from the first line of the note.

Reply:
My bad, I hadn't seen it was used somewhere else ! I'll remove these changes